backend/routers/controllers/message_controllers/topic-like-service.js

```diff
@@ -1,34 +1,70 @@
 const topicLikeModel = require('../../../db/models/message_models/topic-likes-model.js');
 const topicModel = require('../../../db/models/message_models/topic-model.js');
 const topicService = require('./topic-service.js');
+const sequelize = require('sequelize');
+const sequelizeOperators = sequelize.Op;
+
+const moment = require('moment');
+
 function findLikes(userLogin) {
   return topicLikeModel.findAll({
     where: {
       user_login: userLogin,
     },
   });
 }
+function findMonthlyOrWeeklyTopicLikes(searchingType, searchingValue, topics, i) {
+  return topicLikeModel.findAll({
+    raw: true,
+    where: {
+      topic_id: topics[i].id,
+      date: {
+        [sequelizeOperators.gte]: moment().subtract(searchingValue, searchingType).toDate(),
+      },
+    },
+  });
+}
+function isLikesExist(Likes) {
```

Comment:
Does. And place private/helper methods below the main/public method where they are used